OXRS-SHA-StateMonitor-ESP32-FW.ino

```diff
@@ -173,6 +193,7 @@ void initialiseMqtt(byte * mac)
 
 void mqttCallback(char * topic, byte * payload, int length) 
 {
+  screen.trigger_mqtt_rx_led ();

```

Comment:
Again, I am wondering if we would be better off passing a pointer to the screen (optionally) into the MQTT library and let it handle the RX/TX activity triggers? Then the firmware doesn't need to worry.

Reply:
This would allow you to display the MQTT topic inside the MQTT library when you setup the pointer to the screen - again, simplifying the library API.